instrumentation/spring-amqp/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<groupId>io.zipkin.brave</groupId>
+		<artifactId>brave-instrumentation-parent</artifactId>
+		<version>4.14.4-SNAPSHOT</version>
```

Comment:
rebase over master to get latest version, though I don't think anything you are doing is affected by recent versions